acceptance/tests/windows/stop_puppet_svc_stops_puppet.rb

```diff
@@ -0,0 +1,34 @@
+test_name '(C14941) Stopping puppet service on Windows stops puppet'
+
+confine :to, :platform => 'windows'
+
+agents.each do
+  step 'Make sure puppet service is running' do
+    # returns 32 if service is already running
+    result = on(agent, 'sc start puppet', :acceptable_exit_codes => [0, 32])
+    if result.stdout.include?("START_PENDING")
+      sleep 5
+    end
```

Comment:
I think there's a beaker helper to retry logic until a certain condition is true or a timeout is reached. /cc @er0ck